### PR TITLE
Fix indexed values of selectable attributes

### PR DESCRIPTION
**Pull request by JanPetr**

### CODE REVIEW

File: code/Helper/Entity/Producthelper.php [hunk 1]
```diff
@@ -859,7 +860,7 @@ public function getObject(Mage_Catalog_Model_Product $product)
                     if ($customData['in_stock'] && $all_sub_products_out_of_stock) {
                         $customData['in_stock'] = 0;
                     }
-                } elseif ($value === null) {
+                } elseif (!is_array($value)) {
```
Comment (maxiloc) [inline code/Helper/Entity/Producthelper.php]: We might want an OR there, not sure it's an array everytime ?

Comment (maxiloc) [inline code/Helper/Entity/Producthelper.php]: The format is different depending on the attribute type

Comment (JanPetr) [inline code/Helper/Entity/Producthelper.php]: Well, yes, but all other format will pass except `array`. So even `NULL` will go through. Only `array` won't.

Comment (maxiloc) [inline code/Helper/Entity/Producthelper.php]: OK it make sense then

**Pull request merged**